cms/djangoapps/contentstore/management/commands/export_content_library.py

```diff
@@ -0,0 +1,54 @@
+"""
+Script for exporting a content library from Mongo to a tar.gz file
+"""
+from __future__ import print_function
+import os
+
+from django.core.management.base import BaseCommand, CommandError
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+
+from cms.djangoapps.contentstore import tasks
+
+
+class Command(BaseCommand):
+    """
+    Export the specified content library into a directory.  Output will need to be tar zxcf'ed.
+    """
+    help = 'Export the specified content library into a directory'
+
+    def add_arguments(self, parser):
+        parser.add_argument('library_id')
+        parser.add_argument('output_path')
```

Comment:
I think the default value of `output_path` should be `.` to cover the most common use case.

Reply:
👍